### PR TITLE
Added exception handling in MathOperations

### PR DESCRIPTION
**Pull request by Shubhang-2111**
Made changes as per the feedback also added excetion handling in case of division with zero.

### CODE REVIEW

File: src/main/scala/shubhang/TGTraining.scala [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
-  def welcome() = "Welcome to Scala"
+  def subtract(): Int = {
+    num1 - num2
+  }
+
+  def multiply(): Int = {
+    num1 * num2
+  }
+
+  def divide(): Any = {
```
Comment (tg-trainer) [inline src/main/scala/shubhang/TGTraining.scala:17]: never use any as we talked

**Review by tg-trainer — CHANGES_REQUESTED** [review]
Do the required changes

**Review by Shubhang-2111 — COMMENTED** [review]
Made the return type to Int from any.

File: src/main/scala/shubhang/TGTraining.scala [hunk 2]
```diff
@@ -1,6 +1,25 @@
 package shubhang
+import scala.util.{Success,Failure, Try}
 
-class TGTraining {
+class TGTraining(num1:Int,num2:Int) {
```
Comment (tg-trainer) [inline src/main/scala/shubhang/TGTraining.scala:4]: This should be param for your functions otherwise you need to make new object for each new value

**Review by tg-trainer — CHANGES_REQUESTED** [review]

**Review by Shubhang-2111 — COMMENTED** [review]
Passed the params in function instead of class as per the feedback

**Pull request merged**